Sources/Layers/Concat.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  Concat.swift
+//  Bender
+//
+//  Created by Diego Ernst on 6/27/17.
+//
+//
+
+import MetalPerformanceShaders
+
+open class Concat: NetworkLayer {
+
+    var pipeline: MTLComputePipelineState!
+    let axis: LayerSizeAxis
+
+    let maxInputTextures = 10 // must match kernel max input textures
```

Comment:
Actually this is a constant that matches another compiler-constant in the metal shaders. I had no better way to do this